app/lib/features/chat/providers/chat_providers.dart

```diff
@@ -106,3 +107,9 @@ final chatMentionsProvider =
   }
   return mentionRecords;
 });
+
+// state of typing notice setting
+final chatTypingEventStateProvider = StateNotifierProvider.autoDispose<
+    ChatTypingEventStateNotifier, TypingEvent?>(
+  (ref) => ChatTypingEventStateNotifier(ref),
```

Comment:
we should be checking here whether it is enabled via a `ref.watch` to make sure we update it properly